.github/workflows/benchmarks.yml

```
@@ -250,12 +248,13 @@ jobs:
 
             # Show baseline metadata
             echo "=== Baseline Information (from artifact) ==="
-            head -n 3 benches/baseline_results.txt
-          else
-            echo "❌ Downloaded artifact but no baseline_results.txt found"
-            echo "BASELINE_EXISTS=false" >> "$GITHUB_ENV"
-            echo "BASELINE_SOURCE=missing" >> "$GITHUB_ENV"
-          fi
+            head -n 3 baseline-artifact/baseline_results.txt
+        else
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 252: could not find expected ':'

(syntax-check)